lib/src/code_generator/typealias.dart

```diff
@@ -122,3 +135,30 @@ class Typealias extends BindingType {
   String? getDefaultValue(Writer w, String nativeLib) =>
       type.getDefaultValue(w, nativeLib);
 }
+
+/// Objective C's instancetype.
+///
+/// This is an alias for an NSObject* that is special cased in code generation.
```

Comment:
Maybe explain the special casing.

Add a reference to the Objective-C documentation for `instancetype`.